### DISCUSSION TITLE
Abschnitte mit/ ohne Autobahn

### DISCUSSION THREAD

**Discussion by 1maxi2** [devemux86/cruiser#465] [Q&A] [answered]
Gibt es in Cruiser eigentlich die Möglichkeit zum Beispiel für die Anreise die Tour mit Autobahn und den Rest ohne Autobahn zu planen?

**Comment by devemux86** [comment, marked as answer]
@1maxi2 

This is now possible with the new [intelligent route calculation](https://github.com/devemux86/cruiser/discussions/142).
You can change the routing options and select "Recalculate section" and the routing options will only be applied to the route section.

Das ist jetzt mit der neuen [intelligenten Routenberechnung](https://github.com/devemux86/cruiser/discussions/142) möglich.
Sie können die Routing-Optionen ändern und "Abschnitt neu berechnen" wählen, und die Routing-Optionen werden nur auf den Routenabschnitt angewendet.

- https://github.com/devemux86/cruiser/discussions/451

**Reply by 1maxi2** [reply]
O.k. Danke dan warte ich auf die 3.0.41
Könnte man nicht bei Abschnittsprofil zu den Punkten " Direkt / Kein " den Punkt
 " mit/ohne Autobahn " hinzufügen ?
Oder ist " intelligente Routenberechnung " im Prinzip das gleiche ?

**Reply by devemux86** [reply]
@1maxi2 

"Intelligent mode" is better because you can precisely combine many different routing options and the calculation is faster.

Der "Intelligente Modus" ist besser, weil man genau viele verschiedene Routingoptionen kombinieren kann und die Berechnung schneller ist.

**Reply by 1maxi2** [reply]
O.k.

**Comment by umpfmpf** [comment]
Is it then possible to include my favorite profiles (like motorcycles, motorcycles_fast, car...) in the submenu of the section profiles together with the direct line?

**Reply by devemux86** [reply]
@umpfmpf app profiles hold all app options.

What you mean is only the routing options. This is something different.
It can now be done like mentioned here:
- https://github.com/devemux86/cruiser/discussions/451#discussioncomment-8665647

**Comment by 1maxi2** [comment]
Ich bin immer noch der Meinung man sollte sich so nah wie möglich an der Kurviger app 1.Version halten. War meiner ! Meinung noch am verständlichsten.
<img src="https://github.com/devemux86/cruiser/assets/124886589/4ac3c2e9-f3d6-449a-9bd0-8462378c3004" width="50%"/>
